incl/header.html

```diff
@@ -21,6 +21,7 @@
           <li><a href="/download">Get Coq</a></li>
           <li><a href="/documentation">Documentation</a></li>
           <li><a href="/community">Community</a></li>
+          <li><a href="/consortium">Consortium</a></li>
```

Comment:
You should also add a section in the homepage about the Consortium. What is shown on the homepage and what appears in the top bar should be kept in sync.

Reply:
Well, it is not in sync today (no "news" item on the main page), and I'd rather add an entry only once we clarify a few things about the consortium structure with Yves. This page is rather meant for people actively looking for the Consortium (through Google) or for people needing to send a pointer (to their funding agency, for example).

Reply:
The news still have a specific section on the main page (and all the other pages).

Reply:
Ok, but the second point remains.

Reply:
In the meantime I would then add it in some place that is less obvious. But that's up to you.